libs/build_netcdf.sh

```diff
@@ -61,11 +61,20 @@ curr_dir=$(pwd)
 
 LDFLAGS1="-L$HDF5_ROOT/lib"
 LDFLAGS2=$(cat $HDF5_ROOT/lib/libhdf5.settings | grep AM_LDFLAGS | cut -d: -f2)
-[[ $enable_pnetcdf =~ [yYtT] ]] && LDFLAGS4="-L$PNETCDF_ROOT/lib"
+if [[ $enable_pnetcdf =~ [yYtT] ]]; then
```

Comment:
It seems like a contradiction to me if a user is not using MPI but requesting pnetcdf. Should the build process abort in the pnetcdf step?